Auth0/AuthenticationError.swift

```diff
@@ -149,6 +137,8 @@ public struct AuthenticationError: Auth0APIError {
     /// - [timedOut](https://developer.apple.com/documentation/foundation/urlerror/2293002-timedout)
     /// - [internationalRoamingOff](https://developer.apple.com/documentation/foundation/urlerror/2292893-internationalroamingoff)
     /// - [callIsActive](https://developer.apple.com/documentation/foundation/urlerror/2293147-callisactive)
+    ///
+    /// The underlying `URLError` is available in the ``Auth0Error/cause-9wuyi`` property.
```

Comment:
Is `9wuyi` expected? It occurs a few times throughout so I'm guessing so?

Reply:
Yes, it's a unique identifier to point to a specific `cause` property, because it's a protocol (interface) property with a default implementation. Without it, DocC doesn't know whether to link to the definition of the property, or to the default implementation.

Reply:
See https://developer.apple.com/documentation/xcode/formatting-your-documentation-content

<img width="957" alt="Screen Shot 2022-10-28 at 12 40 51" src="https://user-images.githubusercontent.com/5055789/198678022-0eb3cbe5-095d-49c4-815a-157919c437c5.png">

Reply:
Ah neat! Thank you for explaining!